gunicorn/_compat.py

```diff
@@ -110,94 +110,77 @@ def _wrap_error(exc, mapping, key):
     six.reraise(new_err_cls, new_err,
                 exc.__traceback__ if hasattr(exc, '__traceback__') else sys.exc_info()[2])
 
-if PY33:
-    import builtins
-
-    BlockingIOError = builtins.BlockingIOError
-    BrokenPipeError = builtins.BrokenPipeError
-    ChildProcessError = builtins.ChildProcessError
-    ConnectionRefusedError = builtins.ConnectionRefusedError
-    ConnectionResetError = builtins.ConnectionResetError
-    InterruptedError = builtins.InterruptedError
-    ConnectionAbortedError = builtins.ConnectionAbortedError
-    PermissionError = builtins.PermissionError
-    FileNotFoundError = builtins.FileNotFoundError
-    ProcessLookupError = builtins.ProcessLookupError
-
-    def wrap_error(func, *args, **kw):
+import errno
```

Comment:
Sorry, I wasn't clear in my earlier comment. We can drop everything in the `if PY33` branch as well. I added them when I added a copy of berkerpeksag/selectors34 into the Gunicorn codebase. We don't need them since we now use the standard library version of selectors.

Reply:
Yes, that commit looks great!